lib/backend/report_test.go

```diff
@@ -34,6 +34,7 @@ func TestReporterTopRequestsLimit(t *testing.T) {
 		return int(count)
 	}
 
+	requests.Reset()
```

Comment:
Nit: Resetting this just before comparing requests to 0 feels a bit weird... as in it seems now that "reset" function is being tested rather than the fact that requests are initially equals to zero. Maybe do this in `t.Cleanup()`?